### PR TITLE
Update red tile screenshots 

### LINKED ISSUE #2965: Wrong action advised after red tile (PCR test) in FAQs
## Where to find the issue

- https://www.coronawarn.app/en/faq/results/#rat_red_card_which_test "If I see a high-risk encounter in the App, i.e. a red card, should I do a rapid antigen test or PCR-Test?"

red action EN

- https://www.coronawarn.app/de/faq/results/#rat_red_card_which_test "Wenn ich eine hohe Risiko-Begegnung in der App angezeigt bekomme, d. h. eine rote Karte/Kachel, sollte ich dann einen Schnelltest oder PCR-Test machen?"

---
- https://www.coronawarn.app/en/faq/results/#red_card_how_to_test "Red Card? How to go through Corona testing"

red action 2 EN

- https://www.coronawarn.app/de/faq/results/#red_card_how_to_test "Rote Karte? So geht es zum Corona Test"

---
- https://www.coronawarn.app/en/faq/results/#check_in_qr_warning_share "Can this information be voluntarily communicated to the health authority? If so, what information is flowing?"

image

- https://www.coronawarn.app/de/faq/results/#check_in_qr_warning_share "Können diese Informationen freiwillig dem Gesundheitsamt mitgeteilt werden? Wenn ja, welche Informationen fließen?"

---
- https://www.coronawarn.app/en/faq/results/#in_a_nutshell "In a nutshell"

image

- https://www.coronawarn.app/de/faq/results/#in_a_nutshell "Kurzgefasst"

## Describe the issue

Several FAQ articles give advice about what to do after receiving a red tile "Increased Risk" in the app. Some articles (see above) recommend a PCR test.

These articles are in conflict with information posted by the Bundesministerium für Gesundheit. [RKI: Nationale Teststrategie SARS-CoV-2](https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/Teststrategie/Nat-Teststrat.html) links to [BMG: Fragen und Antworten zu COVID-19 Tests](https://www.bundesgesundheitsministerium.de/coronavirus/nationale-teststrategie/faq-covid-19-tests.html), including https://www.bundesgesundheitsministerium.de/coronavirus/nationale-teststrategie/faq-covid-19-tests.html#c23925

image

The regulation [TestV](https://www.gesetze-im-internet.de/coronatestv_2021-10/)<br>"Verordnung zum Anspruch auf Testung in Bezug auf einen direkten Erregernachweis des Coronavirus SARS-CoV-2 (Coronavirus-Testverordnung - TestV)" is now changed and extended in time by the [Dritte Verordnung zur Änderung der Coronavirus-Testverordnung Vom 29. Juni 2022](https://www.bundesanzeiger.de/pub/publication/oLkvcL7aBEpNfC03aH7/content/oLkvcL7aBEpNfC03aH7/BAnz%20AT%2029.06.2022%20V1.pdf?inline). 

## Suggested change

Align the FAQs with the information on  [BMG: Fragen und Antworten zu COVID-19 Tests](https://www.bundesgesundheitsministerium.de/coronavirus/nationale-teststrategie/faq-covid-19-tests.html). Do not recommend a PCR test unless the user has symptoms, in which case the user should contact a doctor. With no symptoms the user should get a Rapid Antigen Test carried out.

## Additional information

The blog https://www.coronawarn.app/en/blog/2021-12-15-cwa-red-tile-guidance/ from Dec 2021 also covers this subject. Blog articles are however not updated and the information accuracy diminishes over time. It should be considered whether this blog article be revised and reissued.

## Related RKI instructions

The [RKI CWA website](https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/WarnApp/Warn_App.html) is outdated concerning actions following a red tile (last updated on /  Stand: 06.05.2022). RKI acknowledged the issue on July 1, 2022. This is however out-of-scope for GitHub issues, since the RKI website is not maintained through GitHub. 

---
Internal Tracking ID: [EXPOSUREAPP-13401](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-13401)

### LINKED ISSUE #3008: Outdated red tile screenshots
### FAQ section

- https://www.coronawarn.app/en/faq/results/#in_a_nutshell
    - https://www.coronawarn.app/assets/img/faq/high_risk_en.png

- https://www.coronawarn.app/de/faq/results/#in_a_nutshell
    - https://www.coronawarn.app/assets/img/faq/high_risk_de.png

### Screenshot section

- https://www.coronawarn.app/en/screenshots/#android_cards
    - https://www.coronawarn.app/assets/screenshots/2-24/en/android/TracingDetailsFragment_tracing_increased_2.png
    - https://www.coronawarn.app/assets/screenshots/2-24/en/android/TracingDetailsFragment_tracing_increased_3.png

- https://www.coronawarn.app/de/screenshots/#android_cards
    - https://www.coronawarn.app/assets/screenshots/2-24/de/android/TracingDetailsFragment_tracing_increased_3.png

- https://www.coronawarn.app/en/screenshots/#ios_cards
    - https://www.coronawarn.app/assets/screenshots/2-24/en/ios/high2en.png

- https://www.coronawarn.app/de/screenshots/#ios_cards
    - https://www.coronawarn.app/assets/screenshots/2-24/de/ios/high2de.png

## Describe the issue

CWA 2.24.1 has changed the advice given with a red tile (increased risk). This means that corresponding screenshots on the cwa-website also need to be updated.

red test guidelines

## Suggested change

Replace the screenshots with ones taken from the CWA 2.24.1 release.

### PR DESCRIPTION
**Pull request by larswmh**
Fixes #3008

See https://github.com/corona-warn-app/cwa-website/pull/3013/files for all screenshot changes. 

---
Internal Tracking ID: [EXPOSUREAPP-13401](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-13401)

### CODE REVIEW

**Comment by MikeMcC399** [comment]
In `#red_card_how_to_test` the embedded image  `/assets/documents/BPA_Corona-Warn-App_RKI-Plakat_Testcenter_A1_ICv2_RZ0` only covers PCR testing. Is there another flow diagram which covers both RAT and PCR testing? If not then the text needs to be adjusted to make it clear if a RAT result can also be shared, etc.

**Comment by MikeMcC399** [comment]
In #rat_red_card_which_test it says "Please confer with your doctor for the further procedure." and this makes it sound like the user should always contact a doctor.

Please make it clear what conditions should prompt to contact a doctor:
- red tile?
- symptoms?
- RAT test result?
- Self-test result?
- Rapid PCR test result?
- PCR test result?

Please check if the following advice is correct and then consider adding it to the text:

_Please consult with a doctor if you are experiencing symptoms or if you had a positive test result from any type of COVID-19 test you had done without a doctor._

File: src/data/faq_de.json [hunk 1]
```diff
@@ -240,7 +240,7 @@
                                 "anchor": "rat_red_card_which_test",
                                 "active": false,
                                 "textblock": [
-                                    "Wenn die Corona-Warn-App Ihnen eine Begegnung mit erhöhtem Risiko (rote Kachel) anzeigt, sollten Sie einen PCR-Test machen und sich in Quarantäne begeben.",
+                                    "Wenn die Corona-Warn-App Ihnen eine Begegnung mit erhöhtem Risiko (rote Kachel) anzeigt, sollten Sie sich mit einem Antigen-Schnelltest testen lassen oder einen Selbsttest durchführen, am besten mehrfach im Abstand von zwei Tagen. Für den Fall, dass ein Testergebnis positiv ausfällt sollten Sie sich durch einen PCR-Test absichern.",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:243]: "ausfällt sollten" => 
"ausfällt, sollten"

(missing comma)

File: src/data/faq_de.json [hunk 2]
```diff
@@ -1233,7 +1233,7 @@
                                 "anchor": "red_card_how_to_test",
                                 "active": false,
                                 "textblock": [
-                                    "Wenn die Corona-Warn-App Ihnen mit der roten Karte ein hohes Infektionsrisiko anzeigt, isolieren Sie sich bitte zügig und konsultieren Sie einen Arzt, um ggf. einen PCR-Test durchführen zu lassen.",
+                                    "Wenn die Corona-Warn-App Ihnen mit der roten Karte ein hohes Infektionsrisiko anzeigt, lassen Sie sich bitte mit einem Antigen-Schnelltest testen oder testen Sie sich selbst, am besten mehrfach im Abstand von zwei Tagen. Für den Fall, dass ein Testergebnis positiv ausfällt sollten Sie sich durch einen PCR-Test absichern.",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:1236]: "ausfällt sollten" => 
"ausfällt, sollten"

(missing comma)

**Comment by larswmh** [comment]
FAQ text changes are now handled in #3014

**Review by dsarkar — APPROVED** [review]

**Pull request merged**